_studio/mfx_lib/encode_hw/h264/include/mfx_h264_encode_hw_utils.h

```diff
@@ -1209,6 +1249,9 @@ namespace MfxHwH264Encode
         CmSurface2DUP *m_wsGpuImage;
         SurfaceIndex  *m_wsIdxGpuImage;
         mfxU8         *m_Yscd;
+
+        BRCFrameParams  m_brcFrameParams;
+        mfxBRCFrameCtrl m_brcFrameCtrl;
```

Comment:
Both `mfxBRCFrameParam` & `mfxBRCFrameCtrl` are part of BRC API (mfxbrc.h). Why we re-define one as `BRCFrameParams` (in case of `MFX_EXT_BRC_DISABLE`) but don't do the same for `BRCFrameCtrl`? We either re-define both or none.